### PR TITLE
chore: add js contract migrate example

### PR DESCRIPTION
**Pull request by fospring**

### CODE REVIEW

File: docs/2.build/2.smart-contracts/release/upgrade.md [hunk 1]
```diff
@@ -146,6 +146,12 @@ Imagine you have a Guest Book where you store messages, and the users can pay fo
 to be "premium". You keep track of the messages and payments using the following state:
 
 <CodeTabs>
+  <Language value="js" language="js">
+    <Github fname="index.js"
+          url="https://github.com/fospring/near-sdk-js/blob/feat-migrate-example/examples/src/basic-updates-base.js"
```
Comment (ailisp) [inline docs/2.build/2.smart-contracts/release/upgrade.md:151]: Maybe point to near/near-sdk-js?

Comment (fospring) [reply docs/2.build/2.smart-contracts/release/upgrade.md:151]: > Maybe point to near/near-sdk-js?

yep,after this PR https://github.com/near/near-sdk-js/pull/400 merged, this link need to point to near/near-sdk-js

Comment (fospring) [reply docs/2.build/2.smart-contracts/release/upgrade.md]: link updated to near organization

**Review by ailisp — APPROVED** [review]
Looks good!

**Pull request merged**